### PR TITLE
Printer demo: wraps strings in `@tr()`

### PR DESCRIPTION
**Pull request by ogoffart**
(doesn't include the native code, only .slint)

### CODE REVIEW

File: examples/printerdemo/ui/printer_queue.slint [hunk 1]
```diff
@@ -159,8 +159,8 @@ component NarrowPrintQueueElement inherits Rectangle {
                 Text {
                     // TODO: text-transform: uppercase
                     text: {
-                        if (root.queue-item.status == "PRINTING") {
-                        "\{root.queue-item.progress}% - \{root.queue-item.status}"
+                        if (root.queue-item.progress > 0) {
```
Comment (ogoffart) [inline examples/printerdemo/ui/printer_queue.slint:162]: This change is what was not purely mechanical

**Review by ogoffart — COMMENTED** [review]
First commit just add @tr

Second commit also fix a few strings.

File: examples/printerdemo/ui/printer_queue.slint [hunk 2]
```diff
@@ -159,8 +159,8 @@ component NarrowPrintQueueElement inherits Rectangle {
...
+                            @tr("{}% - {}",root.queue-item.progress,  root.queue-item.status)
```
Comment (tronical) [inline examples/printerdemo/ui/printer_queue.slint:163]: This calls for context :)

**Review by tronical — APPROVED** [review]
Soon we need an XXX built-in language to find untranslated strings :)

File: examples/printerdemo/ui/printer_queue.slint (hunk 2 above)
Comment (ogoffart) [reply examples/printerdemo/ui/printer_queue.slint:163]: Either context, or named argument

Somehing like one of:

```
 @tr("{progress}% - {status}",  progress: root.queue-item.progress,  status: root.queue-item.status);
 @tr("{progress}% - {status}",  progress = root.queue-item.progress,  status = root.queue-item.status);
```

**Pull request merged**